test/apiv2/12-imagesMore.at

```diff
@@ -43,8 +43,29 @@ s1=$(jq -r .status <<<"${lines[1]}")
 like "$s1" "mytag: digest: sha256:[0-9a-f]\{64\} size: [0-9]\+" \
      "Push to local registry: second status line"
 
+# Push to local registry using the libpod endpoint with quiet=false...
+t POST "libpod/images/$IMAGE/tag?repo=localhost:$REGISTRY_PORT/myrepo&tag=mytag-1" 201
+t POST "/libpod/images/localhost:$REGISTRY_PORT/myrepo/push?tag=mytag-1" 500 \
+  .error~".*x509: certificate signed by unknown authority"
+t POST "/libpod/images/localhost:$REGISTRY_PORT/myrepo/push?tlsVerify=false&tag=mytag-1&quiet=false" 200 \
```

Comment:
```suggestion
t POST "libpod/images/localhost:$REGISTRY_PORT/myrepo/push?tag=mytag-1" 500 \
  .error~".*x509: certificate signed by unknown authority"
t POST "libpod/images/localhost:$REGISTRY_PORT/myrepo/push?tlsVerify=false&tag=mytag-1&quiet=false" 200 \
```

If you start with slash the test helper assumes a full URL and you would need to provide the version as well.

Reply:
Fixed